server/service/src/sync/test/test_data/master_list.rs

```diff
@@ -43,6 +46,43 @@ const MASTER_LIST_2: (&'static str, &'static str) = (
 }"#,
 );
 
+const MASTER_LIST_3: (&'static str, &'static str) = (
```

Comment:
Probably want to test for multiple order types (Will add when doing integration tests in new branch)